packages/velo-external-db/test/drivers/data_provider_matchers.js

```diff
@@ -1,4 +1,7 @@
 const entitiesWithOwnerFieldOnly = (entities) => expect.arrayContaining(entities.map(e => ({ _owner: e._owner })))
 
-module.exports = { entitiesWithOwnerFieldOnly }
+
+const toggleCase = (str) => str.split('').map(c => c === c.toUpperCase() ? c.toLowerCase() : c.toUpperCase()).join('')
```

Comment:
why do you need string split here ? it should work on a string with spaces.
plus, let's inline this, no need to add a function for that.

Reply:
I did it to change string like 'Yo' to 'yO', but it's not necessary because I'm toggling only the first char. 

I put it into the test instead of the function, but not as inline on the calling to `givenStartsWithFilterFor` because it’s seemed to me not clear enough. WDYT ?